index.html

```diff
@@ -1411,20 +1411,23 @@ <h3>Context Injection</h3>
           </p>
 
           <p>
-When securing a document, if an `@context` property is not provided in the document
-or the Data Integrity terms used in the document are not mapped by
-existing values in the `@context` property, implementations MUST inject or add
-an `@context` property with a value of
-`https://w3id.org/security/data-integrity/v2`.
+When an application is securing a document, if an `@context` property is not
+provided in the document or the Data Integrity terms used in the document are
+not mapped by existing values in the `@context` property, implementations SHOULD
+inject or append an `@context` property with a value of
+`https://w3id.org/security/data-integrity/v2` or one or more contexts with at
+least the same declarations, such as the Verifiable Credential Data Model v2.0
+context (`https://www.w3.org/ns/credentials/v2`).
           </p>
 
           <p>
-Context injection is expected to be unnecessary sometimes, such as when the Verifiable
-Credential Data Model v2.0 context (`https://www.w3.org/ns/credentials/v2`)
-exists as a value in the `@context` property, as that context maps all of the
-necessary Data Integrity terms that were previously mapped by
-`https://w3id.org/security/data-integrity/v2`.
+Implementations that do not desire to use JSON-LD processing MAY choose to not
+include a `@context` declaration at the top-level of the document. If a
```

Comment:
Does this mean that step (2) of [Create Proof (eddsa-jcs-2022)](https://w3c.github.io/vc-di-eddsa/#create-proof-eddsa-jcs-2022) algorithm will be optional too?

>Set `proof.@context` to `unsecuredDocument.@context`.

Reply:
I don't think it's optional, it just results in a no-op because `unsecuredDocument.@context` would be undefined in the cases where it isn't present.

Reply:
I'd be fine to add @dlongley's suggestion if @silverpill wants that clarification. It would be a simple editorial change (don't need to do it in this PR).

Reply:
>If `unsecuredDocument.@context` is present, set `proof.@context` to `unsecuredDocument.@context`

+1

I think such clarification would be helpful

Reply:
@silverpill, applied your (editorial) request to all JCS cryptosuites:

* https://github.com/w3c/vc-di-ecdsa/commit/7c91cb9286d30d302adbec2cdbe16f065a44d197
* https://github.com/w3c/vc-di-eddsa/commit/c728ae3edeec57271a07b2484d2d91353215f9b8

Reply:
I did quite a bit of editorial clean up to the JCS Create Proof and Verify Proof algorithms, you might just want to look at the latest live Editor's drafts instead.